src/main/java/duke/Storage.java

```diff
@@ -44,16 +44,16 @@ public TaskList load() {
                 switch (item[0]) {
                 case "T":
                     cmd = new AddCommand(CommandType.TODO, item[2], "");
```

Comment:
would be good to provide comments within the commands so we know what the value of each element in the array is expected to have. 
It is very difficult to understand and read item[1] item[2] item[3] with no context. and i dont think they are items?